molr-mole-remote/src/main/java/io/molr/mole/remote/rest/MoleWebClient.java

```diff
@@ -35,7 +36,17 @@ public static MoleWebClient withBaseUrl(String baseUrl) {
     public <T> Flux<T> flux(String uri, Class<T> type) {
         return clientResponseForGet(uri, TEXT_EVENT_STREAM)
                 .flatMapMany(response -> response.bodyToFlux(type))
-                .cache();
+                .cache()
+                .onErrorMap(throwable -> {
+                	//TODO consider not depending on netty exceptions
```

Comment:
Consider using /* ... */ type comments for comments that should stay in the code.